kubejs/startup_scripts/Material_Registry.js

```diff
@@ -401,14 +407,14 @@ GTCEuStartupEvents.registry('gtceu:material', event => {
         .element(GTElements.get("draconic_superconductor"))
         .color(0xffffff)
         .iconSet('shiny')
-		.flags(GTMaterialFlags.NO_SMELTING, GTMaterialFlags.NO_SMASHING)
+        .flags(GTMaterialFlags.NO_SMELTING, GTMaterialFlags.NO_SMASHING)
         .cableProperties(2147483647, 4, 0, true)
-	event.create('infinity')
-		.ingot()
-		.color(0x000000)
-		.iconSet("shiny")
-		.flags(GTMaterialFlags.GENERATE_PLATE)
-		.components('5x neutronium');
+    event.create('infinity')
+        .ingot()
+        .element(GTElements.get("infinity"))
+        .color(0xffffff)
+        .iconSet('infinity')
+        .flags(GTMaterialFlags.GENERATE_PLATE)
```

Comment:
```suggestion
    event.create('infinity')
        .ingot()
        .element(GTElements.get("infinity"))
        .color(0xffffff)
        .iconSet('infinity')
        .flags(GTMaterialFlags.GENERATE_PLATE)
        .components('5x neutronium');
```
why did you remove the components? it is intentional that this can be decomposed to 5 neutronium

Reply:
...why?

Reply:
Marked as resolved because the decomposition is useless (infinity requires almost 2 stacks of neutronium per)